### PR TITLE
Grpc proxy example

### PR DESCRIPTION
**Pull request by silischev**
Maybe this will be useful for beginners

### CODE REVIEW

File: proxy/grpc/generate.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+# go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+# go install github.com/fatih/gomodifytags@v1.13.0
+# go install github.com/FZambia/gomodifytype@latest
```
Comment (FZambia) [inline proxy/grpc/generate.sh:6]: That's not needed here since we communicate over Protobuf

File: proxy/grpc/generate.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+protoc -I proxyproto proxyproto/proxy.proto --go_out=./proxyproto --go-grpc_out=./proxyproto
+
+gomodifytype -file proxyproto/proxy.pb.go -all -w -from "[]byte" -to "Raw"
```
Comment (FZambia) [inline proxy/grpc/generate.sh:10]: Also no need

File: proxy/grpc/generate.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+bash generate_tags.sh
```
Comment (FZambia) [inline proxy/grpc/generate.sh:12]: No need, just native Protobuf/GRPC generation will do the job

File: proxy/grpc/proxyproto/raw.go [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package proxyproto
```
Comment (FZambia) [inline proxy/grpc/proxyproto/raw.go:1]: Can also be removed

**Comment by FZambia** [comment]
> Maybe this will be useful for beginners

Definitely will, thanks!!

File: proxy/grpc/generate.sh [hunk 5]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+# go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
+# go install github.com/fatih/gomodifytags@v1.13.0
```
Comment (FZambia) [inline proxy/grpc/generate.sh:5]: No need to replace tags, this is only for JSON

File: proxy/grpc/generate.sh (hunk 1 above)
Comment (silischev) [reply proxy/grpc/generate.sh]: fixed

File: proxy/grpc/generate.sh (hunk 2 above)
Comment (silischev) [reply proxy/grpc/generate.sh]: fixed

File: proxy/grpc/generate.sh (hunk 3 above)
Comment (silischev) [reply proxy/grpc/generate.sh]: fixed

File: proxy/grpc/proxyproto/raw.go (hunk 4 above)
Comment (silischev) [reply proxy/grpc/proxyproto/raw.go]: fixed

File: proxy/grpc/generate.sh (hunk 5 above)
Comment (silischev) [reply proxy/grpc/generate.sh]: fixed

**Comment by FZambia** [comment]
Thank you!

**Pull request merged**